packages/components/src/FormField/FormField.tsx

```diff
@@ -178,10 +178,7 @@ export const FormField = React.forwardRef(
       value,
       validations,
     }: FormFieldProps,
-    ref:
-      | Ref<HTMLInputElement>
-      | Ref<HTMLTextAreaElement>
-      | Ref<HTMLSelectElement>,
+    ref: Ref<HTMLInputElement | HTMLTextAreaElement | HTMLSelectElement>,
```

Comment:
This was written incorrectly and was causing issues within the `InputText` component.